lib/logstash/inputs/redis.rb

```diff
@@ -57,14 +59,33 @@ class LogStash::Inputs::Redis < LogStash::Inputs::Threadable
   config :batch_count, :validate => :number, :default => 1
 
   public
+  # public API
+
+  REDIS_INPUT_POISON_MSG = '}-{ poison message from teardown }-{'.freeze
+
+  # use to store a proc that can provide a redis instance or mock
+  def add_external_redis_builder(builder) #callable
```

Comment:
It would be good to document also the parameters and return values of this methods, examples of the syntax used before can be found https://github.com/elastic/logstash/blob/master/lib/logstash/util/accessors.rb#L60 for example (take into account there the methods description is missing, what is wrong).